tsup.config.ts

```diff
@@ -1,7 +1,8 @@
+import { rm } from 'node:fs/promises'
 import { defineConfig, type Options } from 'tsup'
 
 export const commonOptions = {
-  clean: true,
+  clean: false,
```

Comment:
this can probably be removed, it's false by default 👀

Reply:
ok (I added it explicitly to avoid reverting the flag)